### PR TITLE
Remove obsolete GITHUB_ORG and use new github orgs in debbuild

### PR DESCRIPTION
**Pull request by j-rivero**
Tested with new gz- packages [Build Status](https://build.osrfoundation.org/view/all/job/ign-tools2-debbuilder/92/)

### CODE REVIEW

File: jenkins-scripts/docker/lib/debbuild-base.bash [hunk 1]
```diff
@@ -48,7 +48,7 @@ if ${NIGHTLY_MODE}; then
   if ${USE_REPO_DIRECTORY_FOR_NIGHTLY}; then
     mv ${WORKSPACE}/repo \$REAL_PACKAGE_NAME
   else
-    git clone https://github.com/${GITHUB_ORG}/\$REAL_PACKAGE_NAME -b ${NIGHTLY_SRC_BRANCH}
+    git clone https://github.com/gazebosim/\$REAL_PACKAGE_NAME -b ${NIGHTLY_SRC_BRANCH}
```
Comment (scpeters) [inline jenkins-scripts/docker/lib/debbuild-base.bash:51]: gazebo classic is still on GitHub.com/osrf/gazebo

**Comment by chapulina** [comment]
See https://github.com/gazebo-tooling/release-tools/pull/735 for an alternative that keeps osrf

**Comment by chapulina** [comment]
Oh I guess now this and #735 are doing the same

**Review by chapulina — APPROVED** [review]

**Comment by scpeters** [comment]
sorry I hadn't seen #735

**Comment by chapulina** [comment]
> this isn't ready to merge because it doesn't handle the package renaming

Not sure what you mean, will this fail with the current package names?

**Comment by scpeters** [comment]
> > this isn't ready to merge because it doesn't handle the package renaming
> 
> Not sure what you mean, will this fail with the current package names?

our code currently uses the old org name `ignitionrobotics` and the old repository names `ign-*`. This pull request updates the org name to `gazebosim` but retains the old repository names `ign-*`. I had assumed that this wouldn't work, but I did a quick check, and it looks like https://github.com/gazebosim/ign-cmake does actually forward properly, so I think my previous comment is incorrect

**Comment by chapulina** [comment]
Ahh gotcha! I think this is because the repositories were moved incrementally:

1. https://github.com/ignitionrobotics/ign-cmake
2. https://github.com/gazebosim/ign-cmake
3. https://github.com/gazebosim/gz-cmake

**Pull request merged**